assignments/lambda-classes.js

```diff
@@ -1 +1,191 @@
-// CODE here for your Lambda Classes
+// CODE here for your Lambda Classes
+// #### Person                
+                              
+// * First we need a Person class. This will be our `base-class`
+// * Person receives `name` `age` `location` all as attributes
+// * Person receives `speak` as a method.
+// * This method logs out a phrase `Hello my name is Fred, I am from Bedrock` where `name` and `location` are the object's own attributes
+                                         
+                                         
+class Person {                           
+    constructor(attributes){                 
+        this.name=attributes.name;          
+        this.age-attributes.age;           
+        this.location=attributes.location;
+    }                                    
+    speak(name,location) {               

```

Comment:
```suggestion
    speak() {               
```

Reply:
The example is asking for the information to come from the object, not to have it passed as arguments. The "this" keyword prepended to the variable names will reference the object the function is called on